lib/game.rb

```diff
@@ -16,6 +16,7 @@ def initialize
 
     def make_new_board
         @board = Board.new
+        @current_player = @player
```

Comment:
Great! Now it will always default to the player starting the game when it starts over each time, rather than depend on who won the last game.